oneview/resource_logical_interconnect_group.go

```diff
@@ -1817,7 +1821,7 @@ func resourceLogicalInterconnectGroupRead(d *schema.ResourceData, meta interface
 
 	qualityOfService := make([]map[string]interface{}, 0, 1)
 	qualityOfService = append(qualityOfService, map[string]interface{}{
-		"type":                         logicalInterconnectGroup.QosConfiguration.Type,
```

Comment:
I think gofmt is not running properly . Please run it in local first .

Reply:
formatting is done correctly, you are commenting on the removed line.